test/specs/scroll-transition.test.js

```diff
@@ -53,4 +53,92 @@ describe('<ScrollTransition />', () => {
       expect(instance.getSlideDirection(start, end, isWrapping)).toEqual(-1);
     });
   });
+
+  describe('#getSlideStyles', () => {
+    describe('calculating zoomScale', () => {
+      it('should return zoomScale of 1 for all slides when animation !== zoom , ', () => {
+        const wrapper = mount(
+          <ScrollTransition>
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(1)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and 0.85 as a default for slide index 1 & 2 when animation === zoom but no zoomScale defined ', () => {
```

Comment:
Now that I'm looking at it again.  I think the wording for all these _it_ statements should start with "should return a _scale_ of ...".  

Since `zoomScale` is the name of the prop, it's a little confusing to read:
"should return a zoomScale of 1 if ... zoomScale is [blah]".

But if it said:
"should return a scale of 1 if ... zoomScale is [blah]"

Then I think it's more clear that `zoomScale` is the passed in prop, and `scale` is the calculated result.  Does that make sense?

Reply:
Yes, that makes sense.

Reply:
Fixed